crates/metadata-server/src/raft/server.rs

```diff
@@ -774,17 +774,7 @@ impl Member {
 
         loop {
             tokio::select! {
-                Ok(()) = nodes_config_watch.changed() => {
-                    let nodes_config = nodes_config.live_load();
-                    if self.should_leave(nodes_config) {
-                        break;
-                    }
-
-                    self.update_node_addresses(nodes_config);
-                },
-                _ = self.tick_interval.tick() => {
-                    self.raw_node.tick();
-                },
+                biased;
```

Comment:
Do we still need this to be biased? I'm assuming that the majority of the gain comes from the cheap rejection of requests, so perhaps we can remove it?

Reply:
Perhaps we can test with/without it? I'm approving to unblock but I'm a bit worried about starvation or impact to read tail latencies.

Reply:
Yes, let's test it. What I have seen before is that the metadata leader was changing when bringing up a 5 node cluster. This made me think that the Raft messages weren't processed fast enough so that some nodes started campaigning. With this change, I've seen fewer metadata leader changes.